test-requirements.txt

```diff
@@ -1,3 +1,5 @@
 odoo-test-helper
 vcrpy
 vcrpy-unittest
+odoo-addon-delivery_roulier @ git+https://github.com/OCA/delivery-carrier.git@refs/pull/731/head#subdirectory=setup/delivery_roulier
```

Comment:
@geomer198 Why adding these as they are not in this module dependencies ?

Reply:
Added for fixing broken tests in delivery_roulier module.